### DISCUSSION TITLE
Could we expect no required import of withDefaults in definePropsRefs?

### DISCUSSION THREAD

**Discussion by extrem7** [vue-macros/vue-macros#552] [Ideas]
Writing `import { withDefaults } from 'unplugin-vue-macros/macros' assert { type: 'macro' }` may be annoying. Is it possible to have definePropsRefs without importing withDefaults from macros?

**Comment by sxzz** [comment]
There's an existing macro `withDefaults` in Vue and the macro will be conflicted with the one from Vue Macros. So we have to import manually.

**Reply by extrem7** [reply]
What if we rename our one?

**Reply by sxzz** [reply]
Good idea! Would you like to contribute a PR? 🤝
